File: Placement Sheet/Arrays/RemoveDuplicatesFromSortedArray.java
import java.util.Scanner;

public class RemoveDuplicatesFromSortedArray {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size = input.nextInt();

        int[] arr = new int[size];
        System.out.println("Please enter the array's values in sorted manner.");
        for (int i = 0; i < size; i++) {
            System.out.printf("Enter %d number element: ", i + 1);
            arr[i] = input.nextInt();
        }
        input.close();

        int result = removeDuplicates(arr);
        for (int i = 0; i < result; i++) {
            System.out.print(arr[i] + " ");
        }
    }

    private static int removeDuplicates(int[] arr) {
        int count = 1;
        for (int i = 1; i < arr.length - 1; i++) {
            if (arr[i] != arr[i - 1]) {
                arr[count] = arr[i];
                count++;
            }
        }
        return count;
    }
}

File: Placement Sheet/Arrays/Initial Startups/Python/RemoveDuplicatesFromSortedArray.py
def removeDuplicates(arr) -> int:
    count = 1
    for i in range(1, len(arr) - 1):
        if arr[i - 1] != arr[i]:
            arr[count] = arr[i]
            count += 1

    return count

size = int(input("Enter the size of the array: "))
arr = []
print("..........Please Enter Data In Sorted Order..........")
for i in range(0, size):
    data = int(input("Enter the data: "))
    arr.append(data)

print("Original Array: ", arr)
count = int(removeDuplicates(arr=arr))
for i in range(0, count):
    print(arr[i], end=" ")

